packages/types/src/injects.ts

```diff
@@ -19,6 +19,7 @@ export default {
   ...runtimeTypes,
   ...extrinsicTypes,
   AssetOf: 'u128',
+  Address: 'AccountId',
```

Comment:
Does it work with current rc2 if we add this type?

Reply:
no